boards/arm/mg100/Kconfig.defconfig

```diff
@@ -12,9 +12,6 @@ config MODEM
 config MODEM_HL7800
 	default NETWORKING
 
-config NORDIC_QSPI_NOR
-	default BOOTLOADER_MCUBOOT
-
```

Comment:
Is this snippet in the board `Kconfig.defconfig` actually required? Shouldn't `NORDIC_QSPI_NOR` default to enabled whenever the relevant devicetree compatible is `status=okay` and `FLASH` is enabled (which it would have to be to expose the `NORDIC_QSPI_NOR` symbol in the first place, as `NORDIC_QSPI_NOR` depends on `FLASH`)

If the board won't function without this snippet, I can work to enable including it. The reason I removed this was because it creates a dependency loop with this test. The test adds several dependencies to `CONFIG_BOOTLOADER_MCUBOOT`, to prevent MCUBoot being configured unless a flash driver is present. Since `NORDIC_QSPI_NOR` depends on `BOOTLOADER_MCUBOOT` here, and the test adds a dependency of `FLASH_HAS_DRIVER_ENABLED` to `BOOTLOADER_MCUBOOT`, which causes a dependency loop.

Reply:
Thank you for the explanation. Yes looks like this can be removed!